models/stg_amplitude.yml

```diff
@@ -7,59 +7,59 @@ models:
     columns:
       - name: event_id
         description: A counter that distinguishes events
-          - unique
-          - not_null 
+          - unique 
```

Comment:
This test is failing for me on the source data in our warehouse. Is that expected and are you seeing the same?

<img width="1194" alt="image" src="https://user-images.githubusercontent.com/74217849/194472055-576e4a2e-9a7f-4dd9-a75b-b69e9ed30488.png">